Sources/NIO/Channel.swift

```diff
@@ -278,6 +278,9 @@ public enum ChannelError: Error {
 
     /// A `DatagramChannel` `write` was made with an address that was not reachable and so could not be delivered.
     case writeHostUnreachable
+
+    /// An operation that was inappropriate given the current `Channel` state was attempted.
+    case inappropriateOperationForState
```

Comment:
Does this need adding to the `Equatable` conformance?

Reply:
thanks @ianpartridge indeed